doc/nrf/libraries/others/profiler.rst

```diff
@@ -13,129 +13,201 @@ The output is provided using RTT and can be visualized in a custom Python backen
+  For example:
 
-* ``python3 plot_from_files.py test1``
+  .. parsed-literal::
+     :class: highlight
 
-  Plots events from the dataset that is provided as the command-line argument.
+     python3 plot_from_files.py test1
 
-* ``python3 real_time_plot.py test1``
+  In this command, ``test1`` is the dataset name.
+* :file:`real_time_plot.py` - This script connects to the device using RTT, plots data in real-time, and saves the data.
+  When running the script from the command line, provide the dataset name.
+  For example:
 
-  Connects to the device using RTT, plots data in real-time, and saves the data.
-  As command line arguments, provide a dataset name.
+  .. parsed-literal::
+     :class: highlight
 
-* ``python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged``
+     python3 real_time_plot.py test1
 
-  Combines data from test_p and test_c datasets into one dataset (test_merged).
-  Provides clock drift compensation based on synchronization events: sync_event_p and sync_event_c.
+* :file:`merge_data.py` - This script combines data from ``test_p`` and ``test_c`` datasets into one dataset ``test_merged``.
+  It also provides clock drift compensation based on the synchronization events: ``sync_event_p`` and ``sync_event_c``.
   This enables you to observe times between events for the two connected devices.
-  As command-line arguments, provide names of events used for synchronization for a Peripheral (sync_event_p) and a Central (sync_event_c), as well as names of datasets for: the Peripheral (test_p), the Central (test_c), and the merge result (test_merged).
+  As command-line arguments, provide names of events used for synchronization for a Peripheral (``sync_event_p``) and a Central (``sync_event_c``), as well as names of datasets for the Peripheral (``test_p``), the Central (``test_c``), and the merge result (``test_merged``).
+  For example:
+
+  .. parsed-literal::
+     :class: highlight
+
+     python3 merge_data.py test_p sync_event_p test_c sync_event_c test_merged
 
-Visualization
--------------
 
-When running ``plot_from_files.py`` or ``real_time_plot.py``, the profiled events are visualized in a GUI window.
+Running the backend
+===================
 
-When displaying Event Manager events, submissions are marked as dots.
-Processing of the events is displayed as rectangles, visualizing the processing time.
+To enable and run the custom backend, complete the following steps:
 
-Use the :guilabel:`start/stop` button below the plot to pause or resume real time plot translation.
-Scroll to zoom in or out.
-When paused, scrolling zooms to the cursor location.
-When plotting in real-time, scrolling zooms to the right edge of the plot.
-Use the middle mouse button to mark an event submission or processing for tracking, and to display the event data.
+1. Enable the :kconfig:`CONFIG_PROFILER_NORDIC` Kconfig option.
```

Comment:
I would prefer not to inform about Kconfig options in section about scripts (we mix information about scripts and firmware and that could be misleading I think).

Reply:
Maybe we could add an explicit information that the Python scripts are communication with profiler subsystem running on the device to get data from the device in runtime?